proto/redpanda/api/dataplane/v1alpha2/topic.proto

```diff
@@ -292,10 +292,7 @@ service TopicService {
     };
   }
   rpc GetTopicConfigurations(GetTopicConfigurationsRequest) returns (GetTopicConfigurationsResponse) {
```

Comment:
Wondering if we also need to rename this from Get to List accordingly @sago2k8 @bojand . One could argue for both cases. It could be argued as one resource (the configuration of a topic which happens to have several properties) or a list of resources (configurations for topics)

Reply:
Good observation. Maybe `ListTopicConfigurations()` is a better choice. Agree it kinda depends on how you look at it. But I do not feel strongly about it.